mapviz/src/mapviz.cpp

```diff
@@ -921,9 +921,26 @@ void Mapviz::SaveConfig()
 
 void Mapviz::ClearHistory()
 {
+  ROS_INFO("made it to loop");
```

Comment:
I'm guessing this was just added for debugging purposes; it should either be `ROS_DEBUG` or removed.
```suggestion
  ROS_DEBUG("Mapviz::ClearHistory()");
```

Reply:
Correct, for debugging.  Will remove.